Add check for avoid segfault when trying instantiate PDORow manually
http://cvs.php.net/viewvc.cgi/php-src/ext/pdo/pdo_stmt.c?r1=1.118.2.38.2.34&r2=1.118.2.38.2.35&diff_format=u

--- a/ext/pdo/pdo_stmt.c
+++ b/ext/pdo/pdo_stmt.c
@@ -18,7 +18,7 @@
   +----------------------------------------------------------------------+
 */
 
-/* $Id: pdo_stmt.c,v 1.118.2.38.2.34 2008/02/26 00:14:04 iliaa Exp $ */
+/* $Id: pdo_stmt.c,v 1.118.2.38.2.35 2008/05/14 12:34:10 felipe Exp $ */
 
 /* The PDO Statement Handle Class */
 
@@ -2711,10 +2711,12 @@ zend_object_handlers pdo_row_object_hand
 
 void pdo_row_free_storage(pdo_stmt_t *stmt TSRMLS_DC)
 {
-	ZVAL_NULL(&stmt->lazy_object_ref);
-	
-	if (--stmt->refcount == 0) {
-		free_statement(stmt TSRMLS_CC);
+	if (stmt) {
+		ZVAL_NULL(&stmt->lazy_object_ref);
+		
+		if (--stmt->refcount == 0) {
+			free_statement(stmt TSRMLS_CC);
+		}
 	}
 }
 
